docs/multitool-usage.md

```diff
@@ -35,10 +35,10 @@ Sarif.Multitool @microsoft/sarif-multitool help
 Sarif.Multitool convert Current.fpr -tool FortifyFpr -output Current.sarif
 
 : Add file contents from analyzed files and snippets from result regions to SARIF
-Sarif.Multitool rewrite Current.sarif --insert "TextFiles,RegionSnippets" --inline
+Sarif.Multitool rewrite Current.sarif --insert "TextFiles,RegionSnippets" --log "Inline"
```

Comment:
>TextFiles,RegionSnippets [](http://example.com/codeflow?start=48&length=24)

This command-line arg is wrong, everything is semicolon delimeted, can you adjust this to 

--insert TextFiles;RegionSnippets

Note that by switching to semicolon as a delimiter, you don't need to enquote args anymore, so switch to this as well:

--log Inline

Finally, if I'm correct here, all this indicates that you didn't actually attempt to execute these command-lines that you are changing. :) For anything you touch, can you do that?

Thanks!

Reply:
Corrected the separator char to ";"

If multiple options used for the parameter value, its doesn't need to be quoted.

If only 1 option is used, still needs to be quoted, otherwise it give the error:
```
ERROR(S):
  Option 'log' is defined with a bad format.
```

Reply:
Looks strange. Maybe related to an idiosyncratic Commandline library behavior? 

Try moving the positional argument `current.sarif` to the end. Does this work? Note the absence of quotation marks.

Sarif.Multitool rewrite --log Inline --insert TextFiles,RegionSnippets Current.sarif

Reply:
Tested the commands I updated both works

Reply:
the same error:

```
C:\repo\github.com\sarif-sdk\bld\bin\AnyCPU_Debug\Sarif.Multitool\netcoreapp3.1>Sarif.Multitool.exe rewrite --insert TextFiles;RegionSnippets --log Inline Current.sarif
SARIF Multitool 4.1.0
c Microsoft Corporation. All rights reserved.

ERROR(S):
  Option 'log' is defined with a bad format.
  A required value not bound to option name is missing.
```

Reply:
I am confused. You say 'both works', you also say 'same error' for both.